src/Common/src/TypeSystem/Ecma/SymbolReader/UnmanagedPdbSymbolReader.cs

```diff
@@ -82,6 +82,11 @@ private void ThrowExceptionForHR(int hr)
 
         static UnmanagedPdbSymbolReader()
         {
+#if PLATFORM_UNIX
```

Comment:
Platform-specific Ifdefs in the compiler prevent us from adding crosscompiling options in the future. This should use the `RuntimeInformation.IsOSPlatform(OSPlatform.Windows)` API to detect Windows at runtime.